pkg/registry/core/endpoint/storage/storage.go

```diff
@@ -59,3 +68,39 @@ var _ rest.ShortNamesProvider = &REST{}
 func (r *REST) ShortNames() []string {
 	return []string{"ep"}
 }
+
+// Get gets endpoints resource by name in the context (whose significance is tenant and namespace).
+// If the target name starts with prefix of "kubernetes-", and the contextual namespace is "default",
+// this target is considered as "k8s alias", a read-only copy of kubernetes endpoints of default namespace
+// in system tenant, and its content returned is based on the system kubernetes resource.
```

Comment:
I believe it will help the code reader and future maintainer if you put the link of the design doc here.

Reply:
ok the link to design proposal will be added here